.github/workflows/use-case-ATX.yaml

```
@@ -15,47 +15,76 @@ jobs:
+        uses: grafana/run-k6-action@a15e2072ede004e8d46141e33d7f7dad8ad08d9d # v1.3.1
         with:
-          filename: test/k6/src/tests/orders-sms.js
-          flags: -e env=${{ vars.ENV }} -e tokenGeneratorUserName=${{ secrets.TOKENGENERATOR_USERNAME }} -e tokenGeneratorUserPwd=${{ secrets.TOKENGENERATOR_USERPASSWORD }} -e smsRecipient=${{ secrets.AUTOMATEDTEST_SMSRECIPIENT }}
-      - name: Build failure report
+          path: test/k6/src/tests/orders-sms.js
+          flags: -e smsRecipient=${{ secrets.AUTOMATEDTEST_SMSRECIPIENT }}
+          inspect-flags: -e altinn_env=${{ matrix.environment }} # Supplies the value for pre-run validation, when env vars / flags are not yet defined
+      - name: Set failure output
+        id: set-failure
         if: failure()
         run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:1:42: Double quote to prevent globbing and word splitting

(shellcheck)